tests/openvino/test_modeling.py

```diff
@@ -1001,6 +1001,9 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
 
     if is_transformers_version(">=", "4.46.0"):
         SUPPORTED_ARCHITECTURES += ("glm", "mistral-nemo", "minicpm3")
+        # openvino 2025.0 required for disabling check_trace
```

Comment:
it can't be tested with check_trace enabled ?

Reply:
yes, check_trace failed for this model due to dynamic logic for selection moe experts, that different from run to run even data are the same